garrysmod/gamemodes/sandbox/gamemode/gui/iconeditor.lua

```diff
@@ -320,11 +320,19 @@ function PANEL:AboveLayout()
 
 	local ent = self.ModelPanel:GetEntity()
 	local pos = ent:GetPos()
-	local campos = pos + Vector( 0, 0, 200 )
+	local ang, campos
+	
+	if (ent:GetSequence() != 0) and (#ent:GetSequenceList() > 1) then

```

Comment:
I don't think this is a good idea. Not all models with sequences are character models.